askup/templates/askup/index.html

```diff
@@ -0,0 +1,101 @@
+                yourself on these questions to make sure you remember the material.
+                If your classmates also create questions on AskUp, quiz yourself
+                on their questions as well.</p>
+        </div>
+        <div class="intro-col col-md-4">
+            <i class="fa fa-university"></i>
+            <h3>Why does it work?</h3>
+            <p>Self-testing has been shown to be one of the most effective learning
+                strategies. Generating questions will require you to recall information,
+                identify the main points, reflect on how it relates to things you’ve
+                previously learned, and generate your own answer—powerful techniques
+                for learning!</p>
+        </div>
+    </div>
+    <!-- QUOTE -->
+    <div class="twrap row">
+        <i class="fa fa-comment-o"></i>
+        <p>"AskUp is a revolutionary educational tool to enhance learning. It empowers
+            learners to create their own questions and answers, employing techniques
+            that have been shown to improve learning and retention."</p>
+        <h4><br>Jonathan S. Hausmann, MD</h4>
+        <p>HARVARD MEDICAL SCHOOL</p>
+    </div>
+    <!-- FOOTER -->
+    <div class="footerwrap row">
+        <div class="col-sm-4">
+            <h4>About</h4>
+            <div class="hline-w"></div>
+            <p>AskUp is a joint venture between investigators at <a href="http://www.bidmc.org"
+                    target="_blank">Beth Israel Deaconess Medical Center</a> (BIDMC)
+                and engineers in the <a href="http://tlt.harvard.edu/" target="_blank">Teaching and Learning Technologies</a>                (TLT) program at Harvard University. Our project was awarded a
+                <a href="http://hilt.harvard.edu/askup" target="_blank">Spark Grant</a>                from the <a href="http://hilt.harvard.edu/">Harvard Initiative for Teaching &amp; Learning (HILT)</a>.</p>
+        </div>
+        <div class="col-sm-4">
+            <h4>Find out More</h4>
+            <div class="hline-w"></div>
+            <p> To learn more about the educational strategies used in this app, including
+                question-generation, spaced repetition, and testing, check out
+                the book <a href="http://www.amazon.com/Make-It-Stick-Successful-Learning/dp/0674729013"
+                    target="_blank"><i>Make it Stick</i> by Peter C. Brown. &nbsp; <i class="fa fa-share-square-o"></i></a></p>
+        </div>
+        <div class="col-sm-4">
+            <h4>Contact Us</h4>
+            <div class="hline-w"></div>
+            <p>
+                125 Mt. Auburn Street<br> Room 524<br> Cambridge, MA 02138<br>                617-871-0040
+                <br>
+                <a href="mailto:support@askup.net">support@askup.net</a><br>
+                <iframe id="twitter-widget-0" scrolling="no" frameborder="0" allowtransparency="true"
```

Comment:
Remove odd scripts

Reply:
should we? because it's an official Askup twitter banner

Reply:
In this case probably not, thanks for the clarification.